test/e2e/upgrade_test.go

```diff
@@ -25,13 +26,19 @@ const (
 	resourceName = "windows-machine-config-operator"
 	// resourceNamespace is the namespace the resources are deployed in
 	resourceNamespace = "openshift-windows-machine-config-operator"
+	// jobName is the name of the job created to test Windows workloads
+	jobName = "windows-workload-tester"
```

Comment:
I suggest naming it for what it is. Something along the lines of `windowsWorkloadTesterJob`.